tardis/visualization/tools/convergence_plot.py

```diff
@@ -0,0 +1,389 @@
+
+    Parameters
+    ----------
+    iterations : int
+        Number of iterations.
+    name : string
+        Name of the colorscale. Defaults to "jet".
+
+    Returns
+    -------
+    colors: list
+    """
+    cmap = mpl.cm.get_cmap(name, iterations)
+    colors = []
+    for i in range(cmap.N):
+        rgb = cmap(i)[:3]
+        colors.append(mpl.colors.rgb2hex(rgb))
+    return colors
+
+
+class ConvergencePlots(object):
+    """
+    Class to create and update convergence plots for visualizing convergence of the
+    simulation.
+
+    Parameters
+    ----------
+    iteration : int
+        iteration number
+    """
+
+    def __init__(self, iterations, **kwargs):
+        self.iterable_data = {}
+        self.value_data = defaultdict(list)
+        self.iterations = iterations
+        self.current_iteration = 1
+        self.luminosities = ["Emitted", "Absorbed", "Requested"]
+        self.plasma_plot = None
+        self.luminosity_plot = None
+
+        if "colorscale" in kwargs:
+            self.colorscale = transition_colors(
+                kwargs["colorscale"], iterations=self.iterations
+            )
+        else:
+            self.colorscale = transition_colors(iterations=self.iterations)
+
+        if "plasma_plot_config" in kwargs:
+            if not isinstance(kwargs["plasma_plot_config"], dict):
+                raise TypeError("Expected dict in plasma_plot_config argument")
+            self.plasma_plot_config = kwargs["plasma_plot_config"]
+
+        if "luminosity_plot_config" in kwargs:
+            if not isinstance(kwargs["luminosity_plot_config"], dict):
+                raise TypeError(
+                    "Expected dict in luminosity_plot_config argument"
+                )
+            self.luminosity_plot_config = kwargs["luminosity_plot_config"]
+
+    def fetch_data(self, name=None, value=None, item_type=None):
+        """
+        This allows user to fetch data from the Simulation class.
+        This data is stored and used when an iteration is completed.
+
+        Parameters
+        ----------
+        name : string
+            name of the data
+        value : string or array
+            string or array of quantities
+        item_type : string
+            either iterable or value
+
+        """
+        if item_type == "iterable":
+            self.iterable_data[name] = value
+        if item_type == "value":
+            self.value_data[name].append(value)
+
+    def create_plasma_plot(self):
+        """
+        Creates an empty plasma plot.
+        The default layout can be overridden by passing plasma_plot_config dictionary in the run_tardis function.
+        """
+        fig = go.FigureWidget().set_subplots(rows=1, cols=2, shared_xaxes=True)
+        fig.add_scatter(row=1, col=1)
+        fig.add_scatter(row=1, col=2)
+
+        fig = fig.update_layout(
+            xaxis={
+                "tickformat": "g",
+                "title": r"$Shell~~Velocity$",
```

Comment:
Change "Shell Velocity" to just "Velocity" and add the units

Reply:
![image](https://user-images.githubusercontent.com/55894364/123802452-f21ec280-d908-11eb-94eb-a7493081013e.png)
This is what I have done. Is this what you had in mind?

Reply:
Yes, this is much better